application-xpoll-api/src/main/java/com/xwiki/xpoll/internal/DefaultXPollManager.java

```diff
@@ -80,7 +80,7 @@
     private Provider<XWikiContext> contextProvider;
 
     @Inject
-    @Named("compactwiki")
+    @Named("local")
```

Comment:
> I think the problem is that when you use the serializer you are passing the wiki name as a second argument rather than the wiki reference.

That was the problem. Passing the reference gives the expected result.